test/doctests/stdouts/31.v1_11.stdout

```diff
@@ -0,0 +1,33 @@
+[ Info: SetupBuildDirectory: setting up build directory.
+[ Info: Doctest: running doctests.
+┌ Error: doctest failure in ~/juliadocs/Documenter/test/doctests/src/NoMeta.jl:6-9
+│ 
+│ ```jldoctest
+│ julia> baz(20)
+│ 40
+│ ```
+│ 
+│ Subexpression:
+│ 
+│ baz(20)
+│ 
+│ Evaluated output:
+│ 
+│ ERROR: UndefVarError: `baz` not defined in `Main`
+│ Suggestion: check for spelling errors or missing imports. No global of this name exists in this module.
```

Comment:
The difference in `31` is straighforward -- Julia now outputs this `Suggestion` with some errors. So I think we can just support version-specific reference files, rather than trying to hack this with regexes.